benchmark/src/jmh/kotlin/kotlinx/benchmarks/protobuf/ProtoBaseline.kt

```diff
@@ -1,10 +1,6 @@
-/*
- * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
- */
+package kotlinx.benchmarks.protobuf
 
-package kotlinx.benchmarks
-
-import kotlinx.serialization.*
+import kotlinx.serialization.Serializable
```

Comment:
Star import, copyright accidentally deleted (in all protobuf-related files)

Reply:
Move refactoring -- the gift that keeps on giving